.env.template

```
@@ -29,6 +29,12 @@ LLM_QUEST_CLAUDE_EXEC_PATH=claude
 LLM_QUEST_CLAUDE_EXEC_MODEL=
 LLM_QUEST_CLAUDE_EXEC_ARGS=
 
+# Langfuse tracing (optional). When set, all LLM calls are traced automatically.
+# Sign up at https://cloud.langfuse.com or self-host.
+LANGFUSE_SECRET_KEY=
+LANGFUSE_PUBLIC_KEY=
+LANGFUSE_HOST=https://cloud.langfuse.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGFUSE_HOST key should go before the LANGFUSE_PUBLIC_KEY key

(UnorderedKey)